docs/design/design-reviews/decision-log/examples/memory/Deployment/Environments.md

```diff
@@ -72,21 +72,21 @@ Changes to this environment are gated by manual approval by your product's leade
 
 ## Environment Variable Group
 
-### Infrastructure Setup (cse-memory-common)
+### Infrastructure Setup (ise-memory-common)
```

Comment:
why does this even have a reference to ISE/CSE, seems like we could probably skip it all together and just call it memory-common for example